src/net/sourceforge/kolmafia/session/EquipmentManager.java

```diff
@@ -42,83 +49,110 @@
 import org.json.JSONObject;
 
 public class EquipmentManager {
-  public static final int NONE = -1;
-
-  // Mutable equipment slots
-  public static final int HAT = 0;
-  public static final int WEAPON = 1;
-  public static final int HOLSTER = 2;
-  public static final int OFFHAND = 3;
-  public static final int CONTAINER = 4;
-  public static final int SHIRT = 5;
-  public static final int PANTS = 6;
-  public static final int ACCESSORY1 = 7;
-  public static final int ACCESSORY2 = 8;
-  public static final int ACCESSORY3 = 9;
-  public static final int FAMILIAR = 10;
+  public enum Slot {
```

Comment:
I feel like we could spin this out into its own file with great benefit to the readability of the two resulting files, but maybe you tried this

Reply:
perhaps I've just looked at these files too long so it's familiarity, but I do like seeing it at the top of `EquipmentManager`

Could move it and all the SLOTS sets out.

Reply:
That's what I was thinking. It's stylistic obviously but it works for me partly because of Slot being used as an enumerated value / class in our scripting world.